prov/gni/include/gnix_av.h

```diff
@@ -42,16 +42,18 @@
  * system.
  */
 
+int _gnix_av_lookup(struct gnix_fid_av *gnix_av, fi_addr_t fi_addr,
+		    struct gnix_address *addr, size_t *addrlen);
+
 /**
- * @brief  lookup the real address based on the input fi_addr
- *	   and the given av.
+ * @brief  Translate fi_addr_t to struct gnix_address.
  *
- * @param[in] int_av	 pointer to a previously allocated gnix_fid_av
+ * @param[in] av         pointer to a previously allocated fid_av
```

Comment:
This doesn't match the parameter name above (gnix_av) or that of the one in gnix_av.c.